go.mod

```diff
@@ -28,13 +28,17 @@ require (
 	github.com/docker/go-units v0.5.0
 	github.com/fahedouch/go-logrotate v0.1.2
 	github.com/fatih/color v1.13.0
+	github.com/fluent/fluent-logger-golang v1.9.0
 	github.com/hashicorp/go-multierror v1.1.1
 	github.com/ipfs/go-cid v0.1.0
 	github.com/ipfs/go-ipfs-files v0.2.0
 	github.com/ipfs/go-ipfs-http-client v0.4.0
 	github.com/ipfs/interface-go-ipfs-core v0.7.0
 	github.com/mattn/go-isatty v0.0.16
+	github.com/mitchellh/mapstructure v1.5.0
+	github.com/moby/buildkit v0.10.1-0.20220907062659-c9a0f4d2de09
```

Comment:
Can we use the latest v0.11 rc?

Reply:
I think we can, are there any significant features or changes in 0.11 that I should take a loook?